CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
-## Unreleased (next version)
+## [Unreleased] (next version)
 
 Please add new unreleased features here.
 
+## [2.0.0] - 2019-07-22
+
+### Changed
+
+ - Supported version of Terraform is 0.12. [#7]
```

Comment:
Please add a link to the bottom of the file:
[#1]: https://github.com/terraform-google-modules/terraform-MODULE/pull/1